sql/binlog.h

```diff
@@ -949,6 +948,8 @@ class MYSQL_BIN_LOG: public TC_LOG
       @retval !=0    Error
   */
   int get_gtid_executed(Sid_map *sid_map, Gtid_set *gtid_set);
+private:
+  void publish_global_status_variables(void) const;
```

Comment:
Ah, "coordinates" then (a pair of name, position), making  it publish_coordinates_for_global_status ?